e2e/fixtures/options.go

```diff
@@ -219,6 +226,9 @@ func (options *FactoryOptions) validateFlags() error {
 		if err != nil {
 			return err
 		}
+
+		fdbVersion, _ := fdbv1beta2.ParseFdbVersion(options.fdbVersion)
+		options.featureOperatorPodIPFamilyV6 = options.featureOperatorPodIPFamilyV6 && fdbVersion.IsAtLeast(fdbv1beta2.Version{Major: 7, Minor: 0, Patch: 0})
```

Comment:
If the configuration is invalid we should throw an error and not silently move forward. Otherwise a user might run a test with an invalid config without noticing that the test setup is configured differently from the expected setup.